uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -730,4 +730,16 @@ public function testNonExpiredToilRequestsAreDeletedAndExpiredToilRequestsNotDel
     $date = date('Y-m-d', strtotime('+1 day'));
     $this->assertEquals($leaveRequestDate->date, $date);
   }
+
+  public function testGetEnabledSicknessAbsenceTypes() {
+    $absenceType1 = AbsenceTypeFabricator::fabricate(['is_sick' => 1]);
```

Comment:
Please add another test to check that this method only returns active/enabled items